src/pkg/signing/signing.go

```diff
@@ -22,23 +22,13 @@ import (
 )
 
 const (
-	keyserverURI = "https://keys.sylabs.io:11371"
+	defaultKeysServer = "https://keys.sylabs.io:11371"
```

Comment:
Can you remove `11371`? We can use https://keys.sylabs.io now, port 443 will work from everywhere (Starbucks with paranoid firewall etc.)